<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page_maintenance</class>
 <widget class="QWidget" name="page_maintenance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>1920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="pushButton_product_4">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>780</y>
     <width>281</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_product_1">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>320</y>
     <width>281</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_product_3_name">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>1140</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Product 3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_product_2_name">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>670</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Product 2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_product_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>320</y>
     <width>281</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_product_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>780</y>
     <width>281</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_product_4_name">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>1140</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Product 4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_product_1_name">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>670</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Product 1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_to_previous_page">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title_maintenance_mode">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>178</y>
     <width>1080</width>
     <height>86</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MAINTENANCE MODE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_machine_id">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>1320</y>
     <width>391</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>machine serial</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_product_1_status">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>630</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_product_2_status">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>630</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_product_3_status">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>1090</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_product_4_status">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>1090</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ui_version">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>1380</y>
     <width>351</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>UI version</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_general_settings">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>40</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>General Settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_product_1_position">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>290</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>P1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_product_2_position">
   <property name="geometry">
    <rect>
     <x>685</x>
     <y>290</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>P2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_product_3_position">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>750</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>P3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_product_4_position">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>750</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>P4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>1300</y>
     <width>641</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <zorder>label_product_3_name</zorder>
  <zorder>label_product_2_name</zorder>
  <zorder>label_product_4_name</zorder>
  <zorder>label_product_1_name</zorder>
  <zorder>pushButton_to_previous_page</zorder>
  <zorder>label_title_maintenance_mode</zorder>
  <zorder>label_machine_id</zorder>
  <zorder>label_ui_version</zorder>
  <zorder>pushButton_general_settings</zorder>
  <zorder>label_product_1_position</zorder>
  <zorder>label_product_2_position</zorder>
  <zorder>label_product_3_position</zorder>
  <zorder>label_product_4_position</zorder>
  <zorder>pushButton_product_1</zorder>
  <zorder>pushButton_product_2</zorder>
  <zorder>pushButton_product_4</zorder>
  <zorder>pushButton_product_3</zorder>
  <zorder>label_product_3_status</zorder>
  <zorder>label_product_1_status</zorder>
  <zorder>label_product_4_status</zorder>
  <zorder>label_product_2_status</zorder>
  <zorder>line</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
